pkg/windows/windows.go

```diff
@@ -417,50 +408,44 @@ func (vm *windows) EnsureRequiredServicesStopped() error {
 	return nil
 }
 
-// ensureServicesAreRemoved ensures that all services installed by WMCO are removed from the instance
-func (vm *windows) ensureServicesAreRemoved() error {
-	// TODO: In this transitional period of migrating services to WICD's responsibility, WICD must be stopped first to
-	//       ensure it does not start any of these stopped services. This code should be removed as part of
-	//       https://issues.redhat.com/browse/WINC-733
-	svc := &service{name: wicdServiceName}
+// ensureServiceIsRemoved ensures that the given service installed by WMCO is removed from the instance
+func (vm *windows) ensureServiceIsRemoved(svc *service) error {
```

Comment:
IMO have this take in a name for ease of use at call sites and create `service` obj within the method as before